storage/range_command.go

```diff
@@ -328,6 +328,10 @@ func (r *Range) EndTransaction(batch engine.Engine, ms *proto.MVCCStats, args *p
 // intended to serve as a sort of caching pre-fetch, so that the requesting
 // nodes can aggressively cache RangeDescriptors which are likely to be desired
 // by their current workload.
+//
+// Note that read consistency specified in the args header is ignored. Reads
+// are always inconsistent, but every attempt is made to resolve encountered
```

Comment:
OK, made the read consistency configurable again. This means that a request to consistent read a range meta record will hit the intent, try to push the txn with RangeLookup=true, do an inconsistent range lookup to find txn, hit intent, and return random old or new value. It will then try to push the txn. If it fails, it will backoff and retry again until it succeeds.